go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/elazarl/go-bindata-assetfs v1.0.0
 	github.com/gogo/protobuf v1.3.1
 	github.com/golang/groupcache v0.0.0-20191227052852-215e87163ea7 // indirect
+	github.com/golang/protobuf v1.3.3 // indirect
```

Comment:
This may be correct, but run a `go mod tidy` just to be sure.

Reply:
ty ty, always forget that